docs/idl_vs_python/python_vs_IDL_fit.rst

```diff
@@ -0,0 +1,48 @@
+###############################
+IDL vs. Python Spectrum Fitting
+###############################
+
+Fit Results
+===========
+
+The surface brightness data in the following spectra are from a southern region of 
+NGC 2023. To learn more about the region of NGC 2023 examined, refer to *The PAH 
+Emission Characteristics of the Reflection Nebula NGC 2023* by Els Peeters et al.
+
+Changes to Default scipack File Used for Fitting
+================================================
+
+Several changes were made to the default package *scipack_ExGal_SpitzerIRSSLLL.ipac*
+when fitting the infrared spectrum of NGC 2023.  Of note, the **amp** values of 
+H2 S(4), H2 S(6) and H2 S(7) were changed to **0**.  Additionally, the **amp_fixed** 
+values of the same features were changed to **True**.
```

Comment:
I think this section would be better formatted as a note.  See https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html.

Reply:
Thank you for the link!  I didn't realize that I could make notes in rst files.  Everything should be fixed now.